nimterop/grammar.nim

```diff
@@ -76,11 +78,13 @@ proc initGrammar() =
 
     if i+1 < gStateRT.data.len and gStateRT.data[i+1].name == "identifier":
       pname = gStateRT.data[i+1].val.getIdentifier(nskParam)
+      doAssert pname.nBl, &"Blank param name for '{fname}', originally '{gStateRT.data[i+1].val}'"
```

Comment:
not DRY; defined gStateRT.data[i+1].val} once

Reply:
That's where the original value before getIdentifier is stored. I'll have to introduce yet another var when its right there.

Reply:
found a better idea here: https://gitter.im/nimgen/Lobby?at=5c4e39b8c2dba5382e9ac5c2 which will simplify all these code surrounding getIdentifier